node/core/chain-selection/src/tree.rs

```diff
@@ -359,6 +359,9 @@ fn apply_ancestor_reversions(
 	// of unviability is only heavy on the first log.
 	for revert_number in reversions {
 		let maybe_block_entry = load_ancestor(backend, block_hash, block_number, revert_number)?;
+		if let Some(block_entry) = &maybe_block_entry {
+			gum::trace!(target: LOG_TARGET, ?revert_number, ?block_entry.block_hash, "Block marked as reverted via scraped on-chain reversions");
```

Comment:
nit: `cargo fmt` does not work for macros, it's not that critical, but it's better to fold the lines manually to fit into 100 chars